src/adapter_covid19/personal_insolvency.py

```diff
@@ -0,0 +1,326 @@
+from dataclasses import dataclass, field
+from typing import Dict, Mapping, MutableMapping, List, Tuple
+
+import numpy as np
+import pandas as pd
+import scipy.stats
+
+from adapter_covid19.constants import START_OF_TIME, DAYS_IN_A_YEAR
+from adapter_covid19.datasources import (
+    Reader,
+    RegionDataSource,
+    RegionSectorAgeDataSource,
+)
+from adapter_covid19.enums import LabourState, Age
+from adapter_covid19.enums import Region, Sector
+
+
+@dataclass
+class PersonalBankruptcyResults:
+    time: int
+    earning: Mapping[LabourState, float]
+    balance: Mapping[LabourState, float]
+    credit_mean: Mapping[LabourState, float]
+    credit_std: float
+    utilisations: Mapping[LabourState, float]
+    personal_bankruptcy: float
+    corporate_bankruptcy: float
+
+
+@dataclass
```

Comment:
Note model implemented differently from the others (using dataclass)

Reply:
So I've left this as it is, because there's no real reason for this to be a dataclass; this is something we've fixed in the next MR we're about to send across.